odoo/exceptions.py

```diff
@@ -53,6 +53,7 @@ class AccessDenied(UserError):
 
         When you try to log with a wrong password.
     """
+    http_status = HTTPStatus.FORBIDDEN  # 403
```

Comment:
401 error? (because credentials are invalid - but the spec says the response should include a www-authenticate header)

Reply:
I know, however some servers don't provide these headers just to distinguish between authentication (401 - to indicate you are not logged in) and authorization (403 - logged in, but no access) issues.
Let's keep it a 403 as per standards.